src/components/IncomingMessageFilter.jsx

```diff
@@ -367,11 +367,7 @@ IncomingMessageFilter.propTypes = {
   assignmentsFilter: type.shape({
     texterId: type.number
   }).isRequired,
-  tags: type.shape({
-    specificTagIds: type.arrayOf(
-      type.shape({ id: type.string, title: type.string })
-    )
-  }),
+  tags: type.arrayOf(type.shape({ id: type.string, title: type.string })),
```

Comment:
Was this a bug in the prop-types? I don't see anything else changing with respect to the shape here